src/pb-decode.js

```diff
@@ -142,10 +142,10 @@ export function decodeNode (bytes) {
   const l = bytes.length
   let index = 0
   /** @type {RawPBLink[]|void} */
-  let links
+  let links = undefined // eslint-disable-line no-undef-init
```

Comment:
added eslint disables because I have `standard` running in my editor which doesn't like this new TS rule